source/codegen/templates/service_helpers.mako

```diff
@@ -185,6 +185,10 @@ ${initialize_standard_input_param(function_name, parameter)}\
   c_type = parameter['type']
   c_type_pointer = c_type.replace('[]','*')
   c_type_underlying_type = common_helpers.get_underlying_type_name(c_type)
+  size = ''
```

Comment:
You don't need `size` anymore. Remove this line and line 190 below.

Reply:
Removed it